RadBot.py

```diff
@@ -183,9 +183,9 @@ def analyseWallets(self, wallets, colors=False):
                 fees_USDC += current_USDC - expected_USDC
                 fees_eXRD += current_eXRD - expected_eXRD
                 total_value_pooled += 2*initial_USDC
-                IL -= (2*np.sqrt(price_change_factor)/(1+price_change_factor) - 1)*initial_USDC
+                IL -= (2 * growth_factor - price_change_factor - 1) * initial_USDC
```

Comment:
Alternative is
```python
                current_price = (self.portfolio.pool_USDC / 1e6) / (self.portfolio.pool_eXRD / 1e18)
                IL += initial_USDC + initial_eXRD * current_price - 2 * expected_USDC
```

Reply:
Agreed on the alternative, not convinced on the one int he code.
I ran the math myself now, code would be correct with changing to expected USDC and putting the factor 2 back.
But no matter, Adam asked me to present this in form of ROI rather than IL as it's a very poorly understood topic.